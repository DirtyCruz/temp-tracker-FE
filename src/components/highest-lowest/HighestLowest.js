import React from 'react';
import PropTypes from 'prop-types';

function HighestLowest({ highestLocation, lowestLocation }) {
  return (
    <>
      <span>
        <h4>Location with highest temp:</h4>
        <h4>{highestLocation}</h4>
      </span>
      <span>
        <h4>Location with lowest temp:</h4>
        <h4>{lowestLocation}</h4>
      </span>
    </>
  );  
}

HighestLowest.propTypes = {
  highestLocation: PropTypes.string.isRequired,
  lowestLocation: PropTypes.string.isRequired
};

export default HighestLowest;

