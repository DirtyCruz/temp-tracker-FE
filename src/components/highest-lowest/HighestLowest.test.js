import React from 'react';
import { shallow } from 'enzyme';
import HighestLowest from './HighestLowest';

describe('HighestLowest component', () => {
  it('renders HighestLowest', () => {
    const wrapper = shallow(<HighestLowest />);
    expect(wrapper).toMatchSnapshot();
  });
});
