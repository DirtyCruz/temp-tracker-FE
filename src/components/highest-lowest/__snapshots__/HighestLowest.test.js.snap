// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HighestLowest component renders HighestLowest 1`] = `
<Fragment>
  <span>
    <h4>
      Location with highest temp:
    </h4>
    <h4 />
  </span>
  <span>
    <h4>
      Location with lowest temp:
    </h4>
    <h4 />
  </span>
</Fragment>
`;
